streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -1271,4 +1279,71 @@ public void process(final ProcessorSupplier<? super K, ? super V> processorSuppl
 
         builder.addGraphNode(streamsGraphNode, processNode);
     }
+
+    @Override
+    public KTable<K, V> toTable() {
+        return toTable(NamedInternal.empty());
+    }
+
+    @Override
+    public KTable<K, V> toTable(Named named) {
+        Objects.requireNonNull(named, "named can't be null");
```

Comment:
This null check is redundant as we check for null in `toTable(Named, Materialized)` anyway -- can be removed